new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1155 @@
+#             Welcome   to   the   VIM   Tutor
+
+Vim is a very powerful text editor that has many commands. Unlike most text
+editors where you use the keyboard to enter text and arrow keys to move your
+cursor, in Vim you use the keyboard to enter commands to manipulate text.
+These commands allow you to express complex thoughts in just a few keystrokes
+to move through text and manipulate it much faster than in most text editors.
+There is a great amount of commands, too many to explain in a tutor such as
+this, but as you will see over the course of this tutorial series these
+commands are not arbitrarily chosen, they form a sort of language with its own
+grammar.
+
+This tutor is designed to describe enough of the commands that you will be able
+to easily use Vim as an all-purpose editor.  It is important to remember that
+it is set up to teach by use. That means you need to do the exercises to learn
+them properly.  If you only read the text, you will soon forget what is most
+important!
+
+NOTE: The commands in the lessons will modify the text, but those changes won't
+be saved.  Don't worry about messing things up; just remember that pressing
+[«<Esc>»](<Esc>) and then [u](u) will undo the latest change.
+
+This tutorial is interactive, and there are a few things you should know.
+Pressing [«<Enter>»](<Enter>) over text highlighted [like this](holy-grail)
+will take you to some relevant help (hopefully), and pressing ? over any word
+will try to do so too.  You can close the help window by pressing «q».
+Sometimes you will be required to modify text like
+
+--->  this here {expect:this here}
+
+Once you have done the changes correctly, the ✗ sign at the left will change to
+✓. We imagine you can already see how neat Vim can be ;) Other times, you'll be
+prompted to run a command (we will explain this later):
+
+~~~ viml
+    :help <Enter>
+~~~
+
+or press a sequence of keys
+
+    <Esc>0f<SPACE>d3wP$P
+
+Text within angle brackets (like <Enter>) describes a key to press instead of
+text to type.
+
+Now, make sure that your Shift-Lock key is NOT depressed and press the «j» key
+enough times to move the cursor so that Lesson 1.1 completely fills the screen.
+
+## Lesson 1.1: MOVING THE CURSOR
+
+** To move the cursor, press the «h», «j», «k», «l» keys as indicated. **
+
+        ← ↓ ↑ →    Hint: The h key is at the left and moves left.
+        h j k l          The l key is at the right and moves right.
```

Comment:
I am not sure about the arrows though, they are not ASCII characters, is there any danger they will be messed up for people not using UTF-8 as their encoding? Or can we safely assume that anyone smart enough to want to use Vim is also smart enough to use UTF-8 everywhere?